fastlane/lib/fastlane/actions/clear_derived_data.rb

```diff
@@ -8,6 +10,15 @@ def self.run(options)
         UI.success("Successfully cleared Derived Data ♻️")
       end
 
+      # Helper Methods
+      def self.xcode_preferences
+        file = File.expand_path("~/Library/Preferences/com.apple.dt.Xcode.plist")
+        if File.exist?(file)
+          return CFPropertyList.native_types(CFPropertyList::List.new(file: file).value)
+        end
+        return ''
```

Comment:
Can you return `nil` here instead of an empty string?

Reply:
Yes, but it creates some complexity that I would like to avoid. I pushed the changes, let me know if you prefer return an empty string or `nil`.